<launch>
    <node name="led_controller" pkg="cyber_is_led_controller" type="leds_node.py" output="screen">
        <param name="leds_topic" value="/leds"/>
        <param name="state_topic" value="/robot_state"/>
        <param name="side_leds" value="5"/>
        <param name="front_leds" value="4"/>
        <param name="back_leds" value="4"/>
        <param name="frequency" value="30"/>
    </node>
</launch>